solc/CommandLineInterface.cpp

```diff
@@ -1351,6 +1384,9 @@ bool CommandLineInterface::assemble(
 
 	for (auto const& src: m_sourceCodes)
 	{
+		// TODO translate from EVM to eWasm if
```

Comment:
Maybe better to do that in assembly stack directly?

Reply:
Yes, I'd prefer to do it over there.